spec/features/admin/admin_views_all_users_spec.rb

```diff
@@ -2,21 +2,16 @@
 
 feature "admin views all users" do
   scenario "successfully" do
-    user = create(:user, admin: true)
-    log_in_as user
+    admin = create(:user, :admin, name: "John", id: 2)
+    user_1 = create(:user, name: "Will", id: 6)
+    user_2 = create(:user, name: "Alice", id: 3)
```

Comment:
There is no reason to set the record id here manually

Reply:
No, def not, I had that there when I was trying to test the sort order - done in sha: 9bd5721